controls_pyqt/gui.py

```diff
@@ -212,24 +213,25 @@ def setup_ui(self):
     def start_ui(self):
         sys.exit(self.app.exec_())
 
-    def on_trigger(self, trigger: str):
+    def on_trigger(self, key: str, pressed: bool):
+        trigger = KeySignal(key, pressed)
         self.comms.add_to_queue(trigger)
 
     def keyPressEvent(self, key_event: QKeyEvent):
         if key_event.key() == Qt.Key_Escape:
             self.exit_program.exit()
 
         if not key_event.isAutoRepeat():
-            for (key, trigger) in self.KEYS_PRESSED:
+            for key in self.KEYS_PRESSED:
                 # print(key, trigger)
                 if key_event.text() == key:
-                    self.on_trigger(trigger)
+                    self.on_trigger(key, True)
 
     def keyReleaseEvent(self, keyevent):
```

Comment:
typo

Reply:
I don't see how it is a typo; I would like you to please explain this to me.